packages/forklift-console-plugin/src/modules/NetworkMaps/views/details/components/MapsSection/MapsSection.tsx

```diff
@@ -156,6 +156,10 @@ export const MapsSection: React.FC<MapsSectionProps> = ({ obj }) => {
     });
   };
 
+  const onClick = () => {
```

Comment:
```suggestion
  const onClick: React.MouseEventHandler<HTMLButtonElement> | undefined = () => {
```

Reply:
We decided that using function type signatures will be used only when the event handler function is **using parameter(s).**  In case that no parameters are anyway used then adding the function type to the signature is not really needed for catching PF 5 changes errors .